app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -2,37 +2,43 @@
   <fieldset ng-disabled="claimDisabled">
     <div ng-show="storageClasses | hashSize" class="form-group">
       <!--storage class-->
-      <label class="required">Storage Classes</label>
+      <label>Storage Class</label>
+      <div>
+      <ui-select
+        ng-if="storageClasses"
+        ng-model="claim.storageClass"
+        theme="bootstrap"
+        search-enabled="true"
+        title="Select a storage class"
+        class="select-role"
+        flex>
+        <ui-select-match placeholder="Select a storage class">
+          <span>
+            {{$select.selected.metadata.name}}
+          </span>
+        </ui-select-match>
+        <ui-select-choices
+          repeat="sclass as sclass in storageClasses | toArray | filter : { metadata: { name: $select.search } } | orderBy: 'metadata.name'">
+          <div>{{ sclass.metadata.name }}
+            <span ng-if="sclass | annotation : 'description'" class="text-muted">
+              <small>&nbsp;-
```

Comment:
Use `&ndash` (or maybe `&mdash;`)

```html
  <small ng-if="sclass | annotation : 'description'" class="text-muted">
     &ndash;
     ...
   </small>
```

If there's not enough margin, you could use


```html
  <small ng-if="sclass | annotation : 'description'" class="text-muted mar-left-sm">
     &ndash;
     ...
   </small>
```